pilot/pkg/bootstrap/server.go

```diff
@@ -765,8 +765,8 @@ func (s *Server) initMCPConfigController(args *PilotArgs) error {
 func (s *Server) initConfigController(args *PilotArgs) error {
 	if len(args.MCPServerAddrs) > 0 || len(s.mesh.ConfigSources) > 0 {
 		s.mcpOptions = &coredatamodel.Options{
-			DomainSuffix:   args.Config.ControllerOptions.DomainSuffix,
-			IncrementalEDS: args.IncrementalEDS,
+			DomainSuffix: args.Config.ControllerOptions.DomainSuffix,
+			Incremental:  args.IncrementalEDS,
```

Comment:
This is going to change in my next commit, since the CMD arg is going to behave differently.